src/transformers/cache_utils.py

```diff
@@ -186,6 +315,135 @@ def from_legacy_cache(cls, past_key_values: Optional[Tuple[Tuple[torch.FloatTens
         return cache
 
 
+class QuantCache(Cache):
```

Comment:
```suggestion
class QuantizedCache(Cache):
```

Reply:
also this is more like a QuantoQuantizedCache no? 😉 We could see other quantization techniques potentially that re-use the QuantizedCache?

Reply:
Yep, we can make naming more clear